packages/core/src/connectivity/scp-cf/user.ts

```diff
@@ -23,6 +22,23 @@ export interface User extends UserData {
   hasScope: (scope: Scope) => boolean;
 }
 
+/**
+ * Extracts the custom attributes in the JWT
+ * @param decodedToken - Token to read the custom attributes
+ * @returns custom attributes added by the xsuaa to the issued JWT.
+ */
+export function customAttributes(
+  decodedToken: DecodedJWT
+): Map<string, string[]> {
+  if (decodedToken[mappingUserFields.customAttributes.keyInJwt]) {
+    return readPropertyWithWarn(
+      decodedToken,
+      mappingUserFields.customAttributes.keyInJwt
+    ) as Map<string, string[]>;
+  }
+  return new Map<string, string[]>();
+}
```

Comment:
I know this was here before, but why is this a `Map`? and is it really a Map?

Reply:
I create a seperate ticket since there is also no test for the method. The ticket is then to add tests and see if we really need this Map<string,string[]> signature.